### PR TITLE
Nits from full re-read

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: auth48/rfc9202.xml [hunk 1]
```diff
@@ -257,7 +257,7 @@ establish a secure channel and convey authorization information
 specific for this communication relationship to the resource server.</t>
       <t><xref target="C-AS-comm" format="default"/> describes how the communication between
       the client (C) and the authorization server (AS) must be secured.
-      Depending on the used CoAP security mode (see also
+      Depending on the CoAP security mode used (see also
```
Comment (obgm) [inline auth48/rfc9202.xml:260]: This seems to be a little awkward to read.

File: auth48/rfc9202.xml [hunk 2]
```diff
@@ -498,7 +498,7 @@ this key. If the authorization server has no certain knowledge that
 the client's key is already known to the resource server, the client's
 public key <bcp14>MUST</bcp14> be included in the access token's <tt>cnf</tt> parameter. If
 CBOR web tokens <xref target="RFC8392" format="default"/> are used (as recommended in
-<xref target="RFC9200" format="default"/>), keys <bcp14>MUST</bcp14> be encoded, as specified in
+<xref target="RFC9200" format="default"/>), keys <bcp14>MUST</bcp14> be encoded as specified in
```
Comment (obgm) [inline auth48/rfc9202.xml:501]: I did not put a comma in here to emphasize the encoding in RFC 9200.

**Review by obgm — COMMENTED** [review]
Two small comments you might want to elaborate on.

File: auth48/rfc9202.xml (hunk 1 above)
Comment (cabo) [reply auth48/rfc9202.xml:260]: It is also English :-)

File: auth48/rfc9202.xml (hunk 2 above)
Comment (obgm) [reply auth48/rfc9202.xml:501]: Yes, you are entirely correct. Sorry, the comma was put in by the RFC editor and was previously not there exactly for the reason you stated.

File: auth48/rfc9202.xml [hunk 3]
```diff
@@ -1046,7 +1046,7 @@ cookie to continue with the handshake. A similar issue exists with the
 unprotected authorization information endpoint when the resource
 server needs to keep valid access tokens for a long time. Adversaries
 could fill up the constrained resource server's internal storage for a
-very long time with interjected or otherwise retrieved valid access
+very long time with intercepted or otherwise retrieved valid access
```
Comment (obgm) [inline auth48/rfc9202.xml:1049]: Sorry, I already had fixed that in commit 5e1b5a5 after your offline comment.

**Pull request merged**